lib/legacy.js

```diff
@@ -0,0 +1,255 @@
+// Originally normalize-package-data
```

Comment:
nit: `legacy.js` I understand this was pulled from normalize-package-data, but what does this do? how will future travelers view this file name? legacy what? can we name this `normalize-package-data.js` or `fix-typos.js` or `normalize-data.js` or anything more descriptive as to what this function does? 🙏